src/abstractdataframe/show.jl

```diff
@@ -54,6 +54,11 @@ ourshow(io::IO, x::Symbol) = ourshow(io, string(x))
 ourshow(io::IO, x::Nothing; styled::Bool=false) = ourshow(io, "", styled=styled)
 ourshow(io::IO, x::SHOW_TABULAR_TYPES; styled::Bool=false) =
     ourshow(io, summary(x), styled=styled)
+function ourshow(io::IO, x::Markdown.MD)
+    r = repr(x)
+    len = min(something(findfirst(c->c=='\n', r), length(r) + 1) - 1, 50)
+    return print(io, len < length(r) - 1 ? first(r, len)*"…" : first(r, len))
```

Comment:
There is still a mix-up of byte and character indexing here:
```
julia> str = "∀"^4 * "1" * "∃"
"∀∀∀∀1∃"

julia> len = findfirst(==('1'), str)
13

julia> length(str)
6

julia> first(str, len)
"∀∀∀∀1∃"

julia> first(str, length(str, 1, len))
"∀∀∀∀1"
```

To be celar:
* `length`, `first` - use character indexing
* `findfirst` uses byteindexing

To fix this store in `len` not the result of `findfirst`, but use `length(str, i, j)` function to transform it to a character index.

Also please add a test that catches this corner case.